RELEASE_NOTES.md

```diff
@@ -13,21 +13,31 @@ v1.0.0-alpha.xx
 
 ### Breaking changes
 
-- Add the (required) `Manager`/`ManagerInterface` `hasCapability`
-  method. A mechanism to query the manager about which particular
-  capabilities it implements.
-  [#1113](https://github.com/OpenAssetIO/OpenAssetIO/issues/1113)
+- Updated `ManagerInterface` to reflect which methods are required and
+  which are optional, default method implementations have been updated:
 
-- Renamed the `Manager`/`ManagerInterface`
-  `PagedRelationshipSuccessCallback` to
-  `RelationshipQuerySuccessCallback` to better reflect the default of
-  pagnation, and reduce ambiguity when relationship creation methods are
-  added in the future.
-  [#1125](https://github.com/OpenAssetIO/OpenAssetIO/issues/1125)
+  - Pre-initialization methods are a no-op or return empty data.
+  - Post-initialization methods raise a `NotImplementedException`.
+
+  Note that in order to suport multi-language plugins, pure virtual/ABC
+  is not used to denote requierd methods in all cases. See the class API
+  documetnation for more details.
+  [#163](https://github.com/OpenAssetIO/OpenAssetIO/issues/163)
+
+- Added the (required) `Manager`/`ManagerInterface` `hasCapability`
+  method. A mechanism to query which particular capabilities a manager
+  implements. This allows a Host to avoid `NotImplementedException`s
+  when a specific capability has not been implemented by any given
+  manager.
+  [#DR022](doc/decisions/DR022-Default-method-implementations.md),
+  [#1113](https://github.com/OpenAssetIO/OpenAssetIO/issues/1113)
 
 - Removed non-paged versions of the `Manager`/`ManagerInterface`
```

Comment:
[consider][minor] There seems to be a slight overuse of ", and" in this note

Reply:
Oh man there is - sorry! How's [this](https://github.com/OpenAssetIO/OpenAssetIO/pull/1157/commits/22aa0026b64e37d6fbfb786900895a6a5bfc40f1)?